snuba/datasets/configuration/generic_metrics/entity_generic_metrics_distributions_friendlier.yaml

```diff
@@ -0,0 +1,60 @@
+schema: v1
+kind: entity
+name: generic-metrics-distributions
+spec:
+  readable_storage: distributions_storage
+  writable_storage: distributions_bucket_storage
+  columns:
```

Comment:
I would suggest you look how the schema for the errors table would look like in this format. That schema is way larger, it may make you consider less verbose alternatives.

Also, a bit of a crazy idea as it would break yaml, but more compact. 
We have a schema parser in the snuba code, that is capable of parsing the output of a describe command in clickhouse 
https://github.com/getsentry/snuba/blob/master/snuba/migrations/parse_schema.py
We use it to compare the schema defined by a migration with the schema from Clickhouse.

So technically we could parse a schema written in the Clickhouse syntax, which is more compact, if you find a reasonable place to store it.